packages/libs/react-ui/src/components/Input/Input.stories.tsx

```diff
@@ -0,0 +1,102 @@
+import { Button } from '../Button/Button';
+import { SystemIcon } from '../Icons';
+import { Stack } from '../Stack/Stack';
+
+import { IInputProps, Input } from './Input';
+
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+
+const meta: Meta<
+  {
+    leftIcon: keyof typeof SystemIcon;
+  } & IInputProps
+> = {
+  title: 'Components/Input',
+  argTypes: {
+    disabled: {
+      control: {
+        type: 'boolean',
+      },
+    },
+    leftIcon: {
+      options: [
+        undefined,
+        ...(Object.keys(SystemIcon) as (keyof typeof SystemIcon)[]),
+      ],
+      control: {
+        type: 'select',
+      },
+    },
+    rightIcon: {
+      options: [
+        undefined,
+        ...(Object.keys(SystemIcon) as (keyof typeof SystemIcon)[]),
+      ],
+      control: {
+        type: 'select',
+      },
+    },
+    leadingText: {
+      control: {
+        type: 'text',
+      },
+    },
+    outlined: {
+      control: {
+        type: 'boolean',
+      },
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj<
+  {
+    leadingText: string;
+    leftIcon: keyof typeof SystemIcon;
+    rightIcon: keyof typeof SystemIcon;
+  } & Omit<IInputProps, 'leftIcon' | 'rightIcon'>
+>;
+
+export const Dynamic: Story = {
+  name: 'Input',
+  args: {
+    leftIcon: undefined,
+    rightIcon: undefined,
+    leadingText: 'Leading',
+    outlined: false,
+  },
+  render: ({ leftIcon, rightIcon, outlined, leadingText, onChange }) => (
+    <Input
+      id="inlineInputStory"
+      leftIcon={SystemIcon[leftIcon]}
+      rightIcon={SystemIcon[rightIcon]}
+      onChange={onChange}
+      placeholder="This is a placeholder"
+      leadingText={leadingText}
+      outlined={outlined}
```

Comment:
I think you may be missing the disabled property?

Reply:
Yes, I missed that. Added 👍